numpy/core/_dtype.py

```diff
@@ -5,9 +5,44 @@
 """
 from __future__ import division, absolute_import, print_function
 
+import sys
+
 import numpy as np
 
 
+_kind_to_stem = {
+    'u': 'uint',
+    'i': 'int',
+    'c': 'complex',
+    'f': 'float',
+    'b': 'bool',
+    'V': 'void',
+    'O': 'object',
+    'M': 'datetime',
+    'm': 'timedelta'
+}
+if sys.version_info[0] >= 3:
+    _kind_to_stem.update({
+        'S': 'bytes',
```

Comment:
Maybe mention the encoding? Not sure we are consistent about that though.

Reply:
As a comment, not part of the stem.

Reply:
The encoding of `bytes`, you mean? I think that's a property of the default `U->S` conversion, not of the S type itself - so the comment doesn't belong here. You can store whatever encoding you like in `S` (or even not text at all).